macosx/fr.lproj/Localizable.strings

```diff
@@ -685,6 +693,9 @@
 /* Preferences -> Network -> port status */
 "Port is open" = "Le port est ouvert";
 
+/* Inspector -> Peers tab -> table row tooltip */
+/* "Port: %@" = ""; */
```

Comment:
The translation was present previously (line 676-677), so why causing a regression by removing it?